phpBB/phpbb/extension/metadata_manager.php

```diff
@@ -229,7 +229,10 @@ public function validate($name = 'display')
 			case 'all':
 				$this->validate('display');
 
-				$this->validate_enable();
+				if (!$this->validate_enable())
+				{
+					throw new \phpbb\extension\exception($this->user->lang('META_FIELD_NOT_SET', $name));
```

Comment:
Do not translate the langage key here: `throw new \phpbb\extension\exception('META_FIELD_NOT_SET', array($name));`